Moderation-Policy.md

```diff
@@ -1,154 +1,288 @@
-* *Post* refers to the content and titles of any issue, pull request, comment or wiki page.
-* *Moderate* refers to the act of modifying the content and title of, or deleting, any Post for the purpose of correcting or addressing Code of Conduct violations.
-* *Remove* refers to the act of removing the configured write (commit) permissions for an individual Collaborator's GitHub account from *all* Node.js GitHub Organization repositories as well as removing the account from the Node.js GitHub Organization membership.
-* *Ban* refers to the act of blocking an individual GitHub account from any further participation in the Node.js GitHub Organization.
+Any alternative Moderation Policy used for a given repository must be included
+in the root directory of the repository using the `Moderation-Policy.md`
+filename. If a repository does not contain a `Moderation-Policy.md` file, then
+this default policy applies.
+
+## Terms
+
+* *Collaborator* refers to any individual with configured write (commit)
+  permissions to any Node.js GitHub organization repository *other than the
+  Moderation Repository*. See [GitHub's access permissions documentation][] for
+  more information.
+* *TSC* refers to the [Node.js Technical Steering Committee][].
+* *CommComm* refers to the [Node.js Community Committee][].
+* *Post* refers to the content and titles of any issue, pull request, comment
+  or wiki page.
+* *Moderate* refers to the act of modifying the content and title of, or
+  deleting, any Post for the purpose of correcting or addressing Code of Conduct
+  violations.
+* *Remove* refers to the act of removing the configured write (commit)
+  permissions for an individual Collaborator's GitHub account from *all*
+  Node.js GitHub Organization repositories as well as removing the account from
+  the Node.js GitHub Organization membership.
+* *Ban* refers to the act of blocking an individual GitHub account from any
+  further participation in the Node.js GitHub Organization. Bans may be
+  *temporary* (24 hours) or *indefinite*.
 * *Requester* refers to an individual requesting Moderation on a Post.
 
-### Grounds for Moderation
-
-Any Post considered to be in violation of the Node.js [Code of Conduct][] is subject to Moderation.
+## Grounds for Moderation
 
-The [Code of Conduct][] calls out the following as specific examples of
-unacceptable behavior:
+Any Post considered to be in violation of the Node.js [Code of Conduct][] is
+subject to Moderation.
 
-* The use of sexualized language or imagery and unwelcome sexual attention or
-advances
-* Trolling, insulting/derogatory comments, and personal or political attacks
-* Public or private harassment
-* Publishing others' private information, such as a physical or electronic
-  address, without explicit permission
-* Other conduct which could reasonably be considered inappropriate in a
-  professional setting
+The Moderation Team is responsible for deciding what constitutes inappropriate
```

Comment:
It might be important to mention that Collaborators may Moderate non-Collaborator Posts at any time, as stated below. 

With that in mind it would seem a "moderation team" might only be applicable when dealing with moderation of a collaborator